pkg/controller/pravegacluster/pravegacluster_controller.go

```diff
@@ -249,9 +253,12 @@ func (r *ReconcilePravegaCluster) deploySegmentStore(p *pravegav1alpha1.PravegaC
 }
 
 func (r *ReconcilePravegaCluster) syncClusterSize(p *pravegav1alpha1.PravegaCluster) (err error) {
-	err = r.syncSegmentStoreSize(p)
-	if err != nil {
-		return err
+	/*We skip calling syncSegmentStoreSize() during upgrade/rollback from version 07*/
+	if !util.IsClusterUpgradingTo07(p) && !r.isRollbackTriggered(p) {
```

Comment:
We should be checking if rollback from 07 is triggered. Here we're checking if any rollback is triggered. Why can't we use "IsClusterRollbackingFrom07()"?

Reply:
I have changed it